modules/cnf-image-based-upgrade-configure-container-storage-image-cleanup.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-image-based-upgrade-shared-container-partition
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-configure-container-storage-image-cleanup_{context}"]
+= Configuring the automatic image cleanup of the container storage disk 
+
+Configure when the {lcao} cleans up unpinned images in the `Prep` stage by setting a minimum threshold for available storage space through annotations.
+The default container storage disk usage threshold is 50%.
+
+The {lcao} does not delete images that are pinned in CRI-O or are currently used.
+The Operator selects the images for deletion by starting with dangling images and then sorting the images from oldest to newest.
```

Comment:
I think "oldest to newest" could be subjective. Someone could possibly view this as a reference to when the image was pulled, when it was most recently used, or when it was built. LCA uses this last one, as it's the only age-related data to which it has access.

The in-repo doc clarifies this by indicating it uses this timestamp:

oldest to newest, **as determined by the image `Created` timestamp**